### PR TITLE
Rel Notes doc text for 4.6.27 release

### PR DESCRIPTION
**Pull request by jeana-redhat**
For release 04 May 2021
Preview: https://deploy-preview-32180--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-6-release-notes.html#ocp-4-6-27

### CODE REVIEW

File: release_notes/ocp-4-6-release-notes.adoc [hunk 1]
```diff
@@ -2852,3 +2852,19 @@ The Insights Operator can now gather `Datahubs` resources from SAP clusters. Thi
...
 To upgrade an existing {product-title} 4.6 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#update-service-overview_updating-cluster-cli[Updating a cluster by using the CLI] for instructions.
+
+[id="ocp-4-6-27"]
+=== RHBA-2021:1427 - {product-title} 4.6.27 bug fix update
+
+Issued: 2021-05-04
+
+{product-title} release 4.6.27 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2021:1427[RHBA-2021:1427] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2021:1428[RHBA-2021:1428] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
```
Comment (maxwelldb) [inline release_notes/ocp-4-6-release-notes.adoc:2863]: Has this first sentence ever raised any translation concerns?

File: release_notes/ocp-4-6-release-notes.adoc [hunk 2]
```diff
@@ -2852,3 +2852,19 @@ The Insights Operator can now gather `Datahubs` resources from SAP clusters. Thi
 ==== Upgrading
 
 To upgrade an existing {product-title} 4.6 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#update-service-overview_updating-cluster-cli[Updating a cluster by using the CLI] for instructions.
+
+[id="ocp-4-6-27"]
+=== RHBA-2021:1427 - {product-title} 4.6.27 bug fix update
+
+Issued: 2021-05-04
+
+{product-title} release 4.6.27 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2021:1427[RHBA-2021:1427] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2021:1428[RHBA-2021:1428] advisory.
```
Comment (maxwelldb) [inline release_notes/ocp-4-6-release-notes.adoc:2861]: Could golf it to:
```suggestion
{product-title} release 4.6.27 is now available. The bug fixes that are included in this update are documented in the link:https://access.redhat.com/errata/RHBA-2021:1427[RHBA-2021:1427] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2021:1428[RHBA-2021:1428] advisory.
```

Grain of salt and all of that, though--it's all fine boilerplate. :smile:

**Review by maxwelldb — APPROVED** [review]
lgtm. Errata links 404, but I assume that's expected.

File: release_notes/ocp-4-6-release-notes.adoc (hunk 2 above)
Comment (jeana-redhat) [reply release_notes/ocp-4-6-release-notes.adoc:2861]: Your suggestion is closer to how it used to be - was changed to this to be _ever-so-slightly_ more accurate :joy:

File: release_notes/ocp-4-6-release-notes.adoc (hunk 1 above)
Comment (jeana-redhat) [reply release_notes/ocp-4-6-release-notes.adoc:2863]: Not that I've heard, though that is a good question.

**Pull request merged**